src/storm-pomdp/modelchecker/BeliefExplorationPomdpModelChecker.cpp

```diff
@@ -1457,6 +1457,12 @@ BeliefExplorationPomdpModelChecker<PomdpModelType, BeliefValueType, BeliefMDPTyp
     return interactiveUnderApproximationExplorer;
 }
 
+template<typename PomdpModelType, typename BeliefValueType, typename BeliefMDPType>
+void BeliefExplorationPomdpModelChecker<PomdpModelType, BeliefValueType, BeliefMDPType>::setFMSchedValueList(
+    std::vector<std::vector<std::unordered_map<uint64_t, ValueType>>> valueList) {
+    this->interactiveUnderApproximationExplorer->setFMSchedValueList(valueList);
```

Comment:
I think the ``this->`` is not necessary here. For consistency with the rest of the code, I'd remove it.

Reply:
Yes agreed I removed it.